packages/lambda/src/models/agreement.js

```diff
@@ -16,10 +16,11 @@
 
 'use strict'
 
-const { DataTypes } = require('sequelize')
+const { DataTypes, Model } = require('sequelize')
+class Agreement extends Model {}
 
 module.exports = (sequelize) => {
-  return sequelize.define('Agreement', {
+  return Agreement.init({
```

Comment:
Our model class is returned by the exported function.  Using the older `sequelize.define('Agreement', {` syntax we were getting a new (but otherwise identical) object as model class each time it was ran.  Defining the class above and using the `Model.init` syntax allows the same object (class) to be returned each time and allowing `instanceof` checks to operate appropriately.